### PR TITLE
Russian Peasant Algorithm - C++

### PR DESCRIPTION
**Pull request by SangeetaMishra**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

(Write your answer here.)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#978

### CODE REVIEW

**Review by iamrajiv — APPROVED** [review]

**Review by vatsalkesarwani12 — APPROVED** [review]
👍🏻

File: C/other/Russian_Peasant_Algorithm.c [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+int main() {
+	int a, b;
+	printf("Enter the first number: ");
+	scanf("%d", &a);
+	printf("\nEnter the second number: ");
+	scanf("%d", &b);
+	int ans = 0;
+	while(b) {
+		if (b%2) {
```
Comment (atarax665) [inline C/other/Russian_Peasant_Algorithm.c]:
```suggestion
		if (b % 2) {
```

File: C/other/Russian_Peasant_Algorithm.c [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+			ans += a;
+		}
+		a*=2;
```
Comment (atarax665) [inline C/other/Russian_Peasant_Algorithm.c]:
```suggestion
		a *= 2;
```

File: C/other/Russian_Peasant_Algorithm.c [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+		b/=2;
```
Comment (atarax665) [inline C/other/Russian_Peasant_Algorithm.c]:
```suggestion
		b /= 2;
```

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by SangeetaMishra** [comment]
@atarax665 I have made the required changes

**Review by atarax665 — APPROVED** [review]

**Pull request merged**